webui/react/src/components/LogViewerTimestamp.tsx

```diff
@@ -445,53 +442,53 @@ const LogViewerTimestamp: React.FC<Props> = ({
   }, [ dateTimeWidth ]);
 
   return (
-    <Page {...props.pageProps} options={logOptions}>
-      <Section
-        bodyBorder
-        bodyDynamic
-        bodyNoPadding
-        filters={FilterComponent && <FilterComponent
-          filter={filter}
-          filterOptions={filterOptions}
-          onChange={setFilter}
-        />}
-        maxHeight>
-        <div className={css.base} ref={baseRef}>
-          <div className={css.container} ref={container}>
-            <VariableSizeList
-              height={listMeasure.height}
-              itemCount={logs.length}
-              itemData={logs}
-              itemSize={getItemHeight}
-              ref={listRef}
-              width='100%'
-              onItemsRendered={onItemsRendered}
-            >
-              {LogViewerRow}
-            </VariableSizeList>
-          </div>
-          <div className={css.scrollTo}>
-            <Tooltip placement="left" title="Scroll to Top">
-              <Button
-                aria-label="Scroll to Top"
-                className={[ css.scrollToTop, css.show ].join(' ')}
-                icon={<Icon name="arrow-up" />}
-                onClick={handleScrollToTop} />
-            </Tooltip>
-            <Tooltip
-              placement="left"
-              title={direction === DIRECTIONS.BOTTOM_TO_TOP ? 'Tailing Enabled' : 'Enable Tailing'}
-            >
-              <Button
-                aria-label="Enable Tailing"
-                className={enableTailingClasses.join(' ')}
-                icon={<Icon name="arrow-down" />}
-                onClick={handleEnableTailing} />
-            </Tooltip>
-          </div>
+    <Section
+      bodyBorder
+      bodyDynamic
+      bodyNoPadding
+      filters={FilterComponent && <FilterComponent
+        filter={filter}
+        filterOptions={filterOptions}
+        onChange={setFilter}
+      />}
+      maxHeight
+      options={logOptions}
```

Comment:
I don't really like where options appear in the page, but don't have a better idea at the moment.

The only alternative that comes to my mind is to move them inside the "filters" area, maybe separated from the "filters" by a border.

<img width="739" alt="Schermata 2021-05-12 alle 11 31 18" src="https://user-images.githubusercontent.com/5413702/117952677-acd11200-b315-11eb-81f2-4268067591ee.png">

Reply:
I think putting them inside the filter area might make that already busy area a bit harder to process, especially when they are actions that don't deal with filtering.

I don't have any great ideas for these buttons either, but one thing that can help with just general positioning is just to right alight these buttons for now.